libr/core/project.c

```diff
@@ -317,6 +317,17 @@ static bool r_core_project_load(RCore *core, const char *prj_name, const char *r
 	} else {
 		ret = r_core_cmd_file (core, rcpath);
 	}
+	char *prj_path = r_file_dirname(rcpath);
+	if (prj_path) {
+		//check if the project uses git
+		if (!(core->prj->rvc = r_vc_git_load (prj_path))) {
```

Comment:
use a local variable instead of modifying the core->prj->rvc until the value != null. it will save some unnecessary dereferences

```
RVC *rvc = r_vc_git_open(prj_path);
if (!rvc) {
 ..
}
```